src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
cards container should have fixed paddings (see 1024px example), bottom the same as top